yabt/builders/golang.py

```diff
@@ -134,7 +141,6 @@ def go_test_tester(build_context, target):
     buildenv_workspace = build_context.conf.host_to_buildenv_path(
         workspace_dir)
     test_cmd = [join(buildenv_workspace, *split(target.name))]
-    test_cmd.append('--test.v')
```

Comment:
What's the effect of removing the --test.v ?

Reply:
Not getting the PASS statements for each test at the end of execution. But I added them in YSettings in resonai2, in the extra_exec_flags, so for us there is no effect, once it gets deployed.
I also added --logtostderr, which was missing so far. See branch with the same name in resonai2.